go/host/node/node.go

```diff
@@ -611,20 +611,7 @@ func (a *Node) storeBlockProcessingResult(result common.BlockSubmissionResponse)
 // Distributes logs to subscribed clients.
 func (a *Node) sendLogsToSubscribers(result common.BlockSubmissionResponse) {
 	for subscriptionID, encryptedLogs := range result.SubscribedLogs {
-		subscriptionIDBytes, err := subscriptionID.MarshalBinary()
-		if err != nil {
-			log.Error("could not marshal subscription ID to bytes. Cause: %s", err)
-		}
-
-		// Due to our reuse of the Geth log subscription API, we have to return the logs as types.Log objects, and not
-		// encrypted bytes. To get around this, we place the encrypted log bytes into the data field of a "fake" log.
-		wrapperLog := types.Log{
```

Comment:
With the new approach, we control what is sent over the subscription, so no more wrapper logs!